README.md

```diff
@@ -1,25 +1,16 @@
-# Go Fn Development Kit (FDK)
-
 [![GoDoc](https://godoc.org/github.com/fnproject/fdk-go?status.svg)](https://godoc.org/github.com/fnproject/fdk-go)
 
-fdk-go provides convenience functions for writing Go fn code
-
-For getting started with fn, please refer to https://github.com/fnproject/fn/blob/master/README.md
-
-# Installing fdk-go
-
-```sh
-go get github.com/fnproject/fdk-go
-```
-
-or your favorite vendoring solution :)
+# Go FDK Documentation
+This is documentation for the Go function development kit (FDK). The Go FDK  provides convenience functions for writing Go Fn code.
```

Comment:
...for the Go function development kit (FDK) which provides...

Reply:
Updated the paragraph and pushed.